mono/mini/main.c

```diff
@@ -100,17 +100,17 @@ probe_embedded (const char *program, int *ref_argc, char **ref_argv [])
 			if (entry_point == NULL)
 				entry_point = aname;
 		} else if (strncmp (kind, "config:", strlen ("config:")) == 0){
-			printf ("c-Found: %s %llx\n", kind, offset);
+			printf ("c-Found: %s %llx\n", kind, (long long)offset);
```

Comment:
We’re on C99, so you can use the cross-platform `PRId64` from `<inttypes.h>` for all of these:

```
printf ("c-Found: %s " PRId64 "\n", kind, offset);
```